hrjobcontract/js/test/services/utils.service.spec.js

```diff
@@ -11,18 +11,21 @@ define([
   'use strict';
 
   describe('utilsService', function () {
-    var $provide, $rootScope, $q, $uibModal, utilsService, apiService, AbsencePeriod, $window;
+    var $httpBackend, $provide, $rootScope, $q, $uibModal, utilsService, apiService,
+      AbsencePeriod, $window;
 
-    beforeEach(module('job-contract', 'leave-absences.models', function (_$provide_) {
+    beforeEach(module('job-contract', 'leave-absences.models', function (_$provide_, $qProvider) {
       $provide = _$provide_;
       $provide.value('$window', {
         location: {
           assign: jasmine.createSpy('spy')
         }
       });
+      $qProvider.errorOnUnhandledRejections(false);
```

Comment:
why is `$qProvider.errorOnUnhandledRejections(false);` being used? I think handling all promise rejections is a good practice and may help uncover issues with the code.

Reply:
sorry, i forgot to document it